.github/workflows/updateVersionsAndCreatePR.yml

```diff
@@ -0,0 +1,85 @@
+name: Update Versions and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (x.x.x)'
+        required: true
+      zowe_version:
+        description: "Zowe Version Tag"
+        default: zowe-v2-lts
+        required: true
+        type: choice
+        options:
+        - zowe-v1-lts
+        - zowe-v2-lts
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt install gh
+
+      - name: Check gh version
+        run: gh --version
+
+      - name: Create Branch
+        run: |
+          version_name="${{ github.event.inputs.version }}"
+          branch_name="update-versions-${version_name}"
+          git checkout -b "$branch_name"
+
+      - name: Get and Change Component Versions
+        id: get_versions
+        run: |
+          components=("perf-timing" "imperative" "cli-test-utils" "secrets-for-zowe-sdk" "core-for-zowe-sdk" "zos-uss-for-zowe-sdk" "provisioning-for-zowe-sdk" "zos-console-for-zowe-sdk" "zos-files-for-zowe-sdk" "zos-logs-for-zowe-sdk" "zosmf-for-zowe-sdk" "zos-workflows-for-zowe-sdk" "zos-jobs-for-zowe-sdk" "zos-tso-for-zowe-sdk" "cli" "cics-for-zowe-cli" "db2-for-zowe-cli" "ims-for-zowe-cli" "mq-for-zowe-cli" "secure-credential-store-for-zowe-cli" "zos-ftp-for-zowe-cli")

```

Comment:
It can be dynamically added in the script if we can evaluate the major version. It is possible to extend a list in bash rather simply.

Reply:
It would be good to make this list dynamic if possible, so that we don't have to remember another place to update the next time we add or remove a package in this list.

The "zowe-versions.yaml" file has all the info that should be needed to make the list dynamic. The [yq](https://mikefarah.gitbook.io/yq/) tool should be preinstalled in the GitHub Actions runner and may also be helpful - for example to get a list of zowe-v2-lts packages:
```shell
yq '.packages | with_entries(select(.value."zowe-v2-lts")) | keys | .[]' zowe-versions.yaml
```

Reply:
@t1m0thyj what about for the manifest workflow?

Reply:
Since the manifest workflow needs a list of repos (instead of all packages), I think its ok to leave it hard-coded for now.